akka-docs/src/main/paradox/typed/distributed-pub-sub.md

```diff
@@ -0,0 +1,93 @@
+# Distributed Publish Subscribe in Cluster
+
+@@@ note
+For the Akka Classic documentation of this feature see [Classic Distributed Publish Subscribe](../distributed-pub-sub.md).
+Classic Pub Sub can be used by leveraging the `.toClassic` adapters until @github[#26338](#26338).
+@@@
+
+@@project-info{ projectId="akka-cluster-typed" }
+
+## Dependency
+
+Until the new Distributed Publish Subscribe API, see @github[#26338](#26338), 
+you can use Classic Distributed Publish Subscribe 
+[coexisting](coexisting.md) with new Cluster and actors. To do this, add following dependency in your project:
+
+@@dependency[sbt,Maven,Gradle] {
+  group="com.typesafe.akka"
+  artifact="akka-cluster-tools_$scala.binary_version$"
+  version="$akka.version$"
+}
+
+Add the new Cluster API if you don't already have it in an existing Cluster application:
+
+@@dependency[sbt,Maven,Gradle] {
+  group=com.typesafe.akka
+  artifact=akka-cluster-typed_$scala.binary_version$
+  version=$akka.version$
+}
+
+## Sample project
+
+Until @github[#26338](#26338), [this simple example]($github.base_url$/akka-cluster-typed/src/test/scala/docs/akka/cluster/typed/DistributedPubSubExample.scala) shows how to use 
+@ref:[Classic Distributed Publish Subscribe](../distributed-pub-sub.md) with the new Cluster API.
+
+### The DistributedPubSub extension
+
+The mediator can either be started and accessed with the `akka.cluster.pubsub.DistributedPubSub` extension as shown below,
+or started as an ordinary actor, see the full Akka Classic documentation @ref:[Clasic Distributed PubSub Extension](../distributed-pub-sub.md#distributedpubsub-extension).
+
+Scala
```

Comment:
since this is only temporary example until we have new api, I think it's fine to skip Java

Reply:
That is what I was thought as well. Not to spend any more time.